extensions/control-plane/store/sql/transfer-process-store-sql/src/test/java/org/eclipse/edc/connector/store/sql/transferprocess/PostgresTransferProcessStoreTest.java

```diff
@@ -171,6 +171,21 @@ void find_queryByLease() {
 
     }
 
+    @Test
+    void find_queryByState() {
```

Comment:
this should stay in the `TestBase` class

Reply:
the test is going to be different for inmem and postgres though, because of the way how the `state` get stored. 
In-mem stores the entire `TransferProcessStates` object, while Postgres persists it as int. Thus, `state = 800` would work in the Postgres case, but not in the inmem case. And yes, we could make it work generically, but that's beyond the scope of this PR.

Reply:
huh... I don't know how I overlooked that. yes, then it makes sense 🤦